assignment01/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head></head>
+
+<body>
+<!-- James Welu 
+assignment01.htm
+INFO2340.WW
+Thoendel
+12/05/2019
+-->
+    <!--
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    -->
+    <script src="js/jquery-3.4.1.min.js"></script>

```

Comment:
It's fine for you to locally host jQuery if you want - although a CDN hosted file would be fine as well - often faster.

Reply:
Okay, I will probably just link to the CDN hosted file in the future.